modules/ROOT/pages/index.adoc

```diff
@@ -20,6 +20,66 @@ With Mule you can:
 * Use error constructs and try scopes for rapid debugging.
 * Isolate the classloader to protect Mule apps from changes to the runtime or connectors.
 
+[[install_deploy_manage]]
+== Installation, Deployment, Management
+
+//TODO? THIS BEGS FOR A DIAGRAM TO SHOW THE RELATIONSHIPS DESCRIBED HERE
+
+For a Mule app to run, it must be deployed into an environment where the Mule
+is installed.
+
+Mule is packaged with the xref:7.3@studio::index.adoc[Studio] IDE and with
+xref:design-center::index.adoc[Design Center] on Anypoint Platform so
+that you can run a Mule app as you design it.
+
+Mule is integrated into xref:runtime-manager::cloudhub.adoc[CloudHub], a
```

Comment:
The Anypoint Runtime Manager allows you to manage your Mule Applications in different deployments Models.  Cloudhub is the fully managed cloud PaaS solution of the Anypoint Platform. You can run your Mule Applications avoiding the provisioning of the infrastructure and Mule Runtimes. Another deployment model is hybrid, in this case, Mule Applications and Runtimes are running in customer datacenters, but managed from the cloud. We can identify different types of hybrid deployments:
Remote Runtimes (AKA, standalone or "naked"), customers start Mule Runtimes in their datacenter but applications can be deployed and managed through Runtime Manager in the cloud. In this case, the customer provisions both the infrastructure and the Mule Runtime. Hybrid PaaS, customers spin up a PaaS on their datacenter and the Anypoint Runtime Manager spin up Mule Applications in it, in this case, customers provision infrastructure for the Applications to run, but Runtime Manager is responsible of spin up Mule Runtimes and guarantees high availability of Mule Applications. We currently support PCF as a hybrid PaaS and we provide a built-in PaaS solution called Runtime Fabric which runs Mule Runtimes in a containerized environment.